### DISCUSSION TITLE
Broader support for media attachments

### DISCUSSION THREAD

**Discussion by lukavdplas** [UUDigitalHumanitieslab/I-analyzer#1432] [Ideas]
**Is your feature request related to a problem? Please describe.**

We currently support image and PDF attachments to documents, but different kinds of corpora may have other needs. Also, the language and iconography used for these attachments in the frontend is really oriented towards newspaper scans.

**Describe the solution you'd like**

Suggestions:

- While we support image attachments, these are always called  "scans" in the interface, but they can be used for other reasons. In a digital newspaper, for instance, the image attachments could be the image that went with the article, rather than a scan of the text.
- Also support audio attachments. Using the browser's native audio player, this shouldn't be too much work.
- Support generic file attachments. Files would have be downloaded to view them, unless we have added frontend support for the MIME type (currently image and PDF files).

**Additional context**

This isn't high-priority until we run into a corpus that requires it.

**Comment by BeritJanssen** [comment]
See also #492
